src/app/views/main-header/settings/ThemeChooserV9.tsx

```diff
@@ -0,0 +1,68 @@
+import { Menu, MenuTrigger, MenuButton, MenuPopover, MenuList, MenuItem } from '@fluentui/react-components';
+import { useAppDispatch, useAppSelector } from '../../../../store';
+import { componentNames, eventTypes, telemetry } from '../../../../telemetry';
+import { PopupsComponent } from '../../../services/context/popups-context';
+import { changeTheme } from '../../../services/slices/theme.slice';
+import { BrightnessHighRegular, WeatherMoonFilled, CircleHalfFillFilled } from '@fluentui/react-icons';
+
+const availableThemes = [
+  {
+    key: 'light',
+    displayName: 'Web Light',
+    icon: <BrightnessHighRegular />
+  },
+  {
+    key: 'dark',
+    displayName: 'Web Dark',
+    icon: <WeatherMoonFilled />
+  },
+  {
+    key: 'high-contrast',
+    displayName: 'Teams High Contrast',
+    icon: <CircleHalfFillFilled />
+  }
+];
+
+const ThemeChooserV9: React.FC<PopupsComponent<null>> = () => {
+  const dispatch = useAppDispatch();
+  const appTheme = useAppSelector(state=> state.theme);
+
+
+  const handleChangeTheme = (selectedTheme: { key: string; displayName: string; icon: JSX.Element }) => {
+    const newTheme: string = selectedTheme?.key ?? '';
+    // Applies the theme to the Fluent UI components
+    switch (newTheme) {
```

Comment:
```suggestion
 dispatch(changeTheme(newTheme))
 loadGETheme(newTheme)   
```

Remove the switch statement because the chooser (in this case menu items) give us defined values.
Use `loadGETheme` to also cater for v8 components. Add a note/todo to remove it later.